### PR TITLE
feat(@angular-ru/eslint-config): add strict-boolean-expressions

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by nikitasadchikov — APPROVED** [review]

**Review by Schirbak — DISMISSED** [review]

**Comment by splincode** [comment]
https://github.com/ng-packagr/ng-packagr/issues/2001

**Comment by splincode** [comment]
https://github.com/angular/angular-cli/issues/21088

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
 
         strategy:
             matrix:
-                node-version: [14.x]
+                node-version: [15.x]
```
Comment (alan-agius4) [inline .github/workflows/workflow.yml:16]: Node.JS 15 is unsupported as odd numbered version of Node.JS are pre-releases and are not considered as stable. 

Please use an LTS version Node.JS 14 (recommended) or the current stable version 16.

https://nodejs.org/en/about/releases

Comment (splincode) [reply .github/workflows/workflow.yml]: done

File: packages/tooltip/src/services/ngx-tooltip-text.interceptor.ts [hunk 2]
```diff
@@ -8,6 +8,6 @@ export class NgxTooltipTextInterceptor implements TooltipTextInterceptor {
     constructor(private readonly translate: TranslateService) {}
 
     public instant(value?: string | null): string | undefined | null {
-        return value ? this.translate.instant(value) : value;
+        return typeof value === 'string' ? this.translate.instant(value) : value;
```
[the remark this answers is not in the document]
Comment (Schirbak) [reply packages/tooltip/src/services/ngx-tooltip-text.interceptor.ts:11]: bump

**Review by Schirbak — APPROVED** [review]

**Review by mansurishmuhametov — APPROVED** [review]

**Review by nikitasadchikov — APPROVED** [review]

**Pull request merged**